vector/src/main/java/im/vector/app/core/platform/VectorBaseFragment.kt

```diff
@@ -200,6 +200,7 @@ abstract class VectorBaseFragment<VB: ViewBinding> : BaseMvRxFragment(), HasScre
     }
 
     protected fun showLoadingDialog(message: CharSequence? = null, cancelable: Boolean = false) {
+        progress?.dismiss()
```

Comment:
We should probably do nothing if `progress` is not null here to avoid blink effect.

Reply:
Ok, the text can change... Ok as a quick fix